### PR TITLE
[JENKINS-69714] Use new jenkins-table for slow trigger admin monitor

### PR DESCRIPTION
**Pull request by kaltepeter**
See [JENKINS-69714](https://issues.jenkins.io/browse/JENKINS-69714).

No tests are included in the pull request because it is difficult to assert table layout from an automated test.

### Before
image

### After
image

### Proposed changelog entries

Replace the old Jenkins table layout in the slow trigger admin monitor with the new Jenkins table layout

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  - Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate.
- [x] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [x] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7205"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy
Comment (kaltepeter) [inline core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy:18]: I added a 'p' tag wrapper to give a bit of space around the text and table. The old one was also crunched and I am not sure if this would align with the desired style. 

Without a class added it takes on a small amount of space based on the theme.

File: core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy
Comment (NotMyFault) [inline core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy:20]:
```suggestion
        table(class: "sortable jenkins-table", width: "100%") {
```
To remove the sharp borders from the table corners, we can drop the `pane` argument.

**Review by NotMyFault — COMMENTED** [review]
Hey,

thanks for your PR.

File: core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy
Comment (kaltepeter) [reply core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy]: Updated. I updated the screen above, unfortunately, I had difficulty re-creating an actual slow trigger so it says 0ms.

**Review by NotMyFault — APPROVED** [review]

File: core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy
Comment (timja) [reply core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy]: you should be able to update the admin monitor to always be enabled and then return some hardcoded data to trigger the issue

**Comment by timja** [comment]
From looking at the screenshots the dismiss button is mis-aligned from the text would be good to fix that. 

(these admin monitor buttons are a mess, a future improvement would be to replace that with an `(x)` button and clean up all the button in general)

**Comment by timja** [comment]
> I am not well versed in the Jenkins theme or classes, or best approaches. If there is guidance here I am happy to use that. I found the class jenkins-!-margin-top-2 by poking through the code

Guidance is here:
https://weekly.ci.jenkins.io/design-library/Spacing/

What you have now looks good enough in respect of current UI, I've created https://issues.jenkins.io/browse/JENKINS-69787 for the enhancement I suggested

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by kaltepeter** [comment]
@timja should I do anything more or be concerned that the auto-merge didn't take this PR after several days?

I can rebase if that is expected, no conflicts yet.

**Comment by NotMyFault** [comment]
>  the auto-merge didn't take

There's no such thing on this repository. Typically, PRs are merged towards weekly releases, which take place every Tuesday, therefore there's no real difference between pulling something now or days in advance.

And no, there's nothing left for you to do :)

**Comment by kaltepeter** [comment]
Thanks, @timja that makes sense. I saw the comment above and thought there was something with automation.

**Comment by timja** [comment]
Apologies that it missed the last weekly, I was away for a week and I would've thought someone else would do the merge 😢

**Comment by NotMyFault** [comment]
> Apologies that it missed the last weekly, I was away for a week and I would've thought someone else would do the merge 😢

The amount of core maintainers merging PRs stagnated a lot, I can count the current ones on a single hand 👀

**Comment by jmMeessen** [comment]
Hello @kaltepeter 

I am Jenkins Developer Relations and promoting Hacktoberfest@Jenkins. I am collecting stories of this year's participants to share with the community. 

First of all, thank you very much for contributing to Jenkins during this Hacktoberfest. It is much appreciated.

This is why I am reaching out to you to better get to know you as a community member and a contributor. Would you mind to answer the following questions?

- Who are you? Where are you located? What do you do for a living?
- Is it the first time you participate to Hacktoberfest?
- What are your motivations?
- Did you already contribute to Open Source before?
- Why is Open Source important to you (as you are contributing)?
- What did you contribute to during this hacktoberfest?
- What do you experience as most difficult when contributing to open source?
- What would you say to others that are hesitating to contribute (because of lack of skills/fear/...)

You can reach me privately via jean-marc@meessen-web.org. 

If it is easier for you I could setup a call during one of your evenings (I am located in Europe) and have a short chat together. I would love to hear from you so that I have a better grasp of the multiple facets of our Community.

If you answer in this comment, by mail (or if we setup a call), would you be OK if I share your story (anonymously or by mentioning your name) with the Community? It would be either during a Hacktoberfest related livestream or a blog post. I strongly believe that sharing real human stories would motivate people to make the step and join us during this month or later.

/- Jmm

Brussels, Belgium
jean-marc@meessen-web.org

**Pull request merged**
